lib/plugins/aws/package/compile/events/alb/index.js

```diff
@@ -101,6 +101,7 @@ class AwsCompileAlbEvents {
         },
         multiValueHeaders: { type: 'boolean' },
         priority: { type: 'integer', minimum: 1, maximum: 50000 },
+        targetGroupName: { type: 'string', minLength: 1, maxLength: 32 },
```

Comment:
Does AWS has any specific naming rules to this name? Are all chars allowed? If we know the allowed charset it might be nice to set `pattern` property

Reply:
Indeed you're right, it should only contain hyphens and alphanumeric, I will add a pattern : https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-name